### ISSUE TITLE
ChatSessionStripRoleName example not working reliably.

### ISSUE THREAD

**Issue by YourNightmar3** [SciSharp/LLamaSharp#375]
I have cloned the repo and when i run the ChatSessionStripRoleName example it doesn't reliably work, sometimes the role name still comes through in the output, it is my understanding that the point of this example is for this not to happen, so this would be a bug. Is this correct?

<img width="1908" alt="image" src="https://github.com/SciSharp/LLamaSharp/assets/28887933/ad1503eb-23bc-4883-9bdd-4d7bdc46d0f3">

<img width="862" alt="image" src="https://github.com/SciSharp/LLamaSharp/assets/28887933/e0f72661-00e1-4000-8f96-a7df9dbfbb46">

I do of course have the `WithOutputTransform` in the code (Since i'm just running the example):
<img width="494" alt="image" src="https://github.com/SciSharp/LLamaSharp/assets/28887933/fc8c43fb-7ab4-461a-ab84-c6d6094f496a">
The only thing i changed in the example is the `GpuLayerCount` to offload more of the model to my GPU and have it run faster.

Is there anything i can do to prevent this from happening? This behavior also occurs in my own test project so it's not just in the example project, and it also happens with different models (the one i am using in the screenshots is nous-hermes-llama2-13b.Q5_K_M.gguf)

**Comment by AsakusaRinne** [comment]
Yes it's a BUG. There has been some modifications about ChatSession recently and it's not stable on master branch. Could you please try the same could with v0.8.1 to see if that still happens?

We'll fix it as soon as possible on master branch.

**Comment by Alword** [comment]
there are also problems with version 0.8.1, he continues to write for the user instead of answering
